File: LayoutTests/http/tests/contentdispositionattachmentsandbox/resources/xml-stylesheet-processing-instructions-frame.php
<?php
header("Content-Disposition: attachment; filename=test.xhtml");
header("Content-Type: application/xhtml+xml");
echo "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n";
echo "<?xml-stylesheet href=\"data:text/css,body::after { content: 'FAIL'; }\" ?>\n";
echo "<html xmlns=\"http://www.w3.org/1999/xhtml\">\n";
echo "<body></body>\n";
echo "</html>\n";
?>

File: LayoutTests/http/tests/contentdispositionattachmentsandbox/resources/xml-stylesheet-processing-instructions-frame.py
#!/usr/bin/env python3

import sys

sys.stdout.write(
    'Content-Disposition: attachment; filename=test.xhtml\r\n'
    'Content-Type: application/xhtml+xml\r\n\r\n'
    '<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n'
    '<?xml-stylesheet href=\"data:text/css,body::after { content: \'FAIL\'; }\" ?>\n'
    '<html xmlns=\"http://www.w3.org/1999/xhtml\">\n'
    '<body></body>\n'
    '</html>\n'
)